src/linearmixedmodel.jl

```diff
@@ -731,16 +735,20 @@ end
 """
     reevaluateAend!(m::LinearMixedModel)
 
-Reevaluate the last column of `m.A` from `m.feterms`.  This function should be called
-after updating the response, `m.feterms[end]`.
+Reevaluate the last column of `m.A` from `m.feterm`.  This function should be called
+after updating the response, `m.fe[end]`.
```

Comment:
There isn't a field `fe`, which ties into another thing I was thinking. Namely, that we should think about making the documentation for the `femat` vs `feterm` field very clear (and also clarify why we need both).

Reply:
Yes, absolutely.  I would still keep open the possibility of not using this change.  I think it will be helpful but it may cause too much pain with the sparse model matrices.  We'll see.  I just wanted to get it to a place where you can look at it.